specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/keys.json

```diff
@@ -1870,6 +1870,24 @@
           "type": "string",
           "format": "base64url",
           "readOnly": true
+        },
+        "iv": {
+          "x-ms-client-name": "result",
+          "type": "string",
+          "format": "base64url",
+          "readOnly": true
+        },
+        "tag": {
+          "x-ms-client-name": "result",
+          "type": "string",
+          "format": "base64url",
+          "readOnly": true
+        },
+        "aad": {
+          "x-ms-client-name": "result",
```

Comment:
I'd call this `additionalAuthenticatedData` unless you have a better suggestion. Generated code would at least have the name we plan to go with.

Reply:
fixed